src/lib.rs

```diff
@@ -159,8 +159,7 @@ async fn new_session(
     payload: Bytes,
 ) -> impl IntoResponse {
     let ttl = sessions.ttl;
-    // TODO: should we use something else? Check for colisions?
```

Comment:
On it's own I don't believe ULID guarantees uniqueness. However, if you  use the monotonic [generator](https://docs.rs/ulid/latest/ulid/struct.Generator.html) capability then it would.

Reply:
Just did that, and added a test which generates 1000 request and check that the returned IDs are monotonically increasing